website/docs/routes/university.md

```diff
@@ -2987,6 +3077,18 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 </Route>
 
+## 四川师范大学 {#si-chuan-shi-fan-da-xue}
+
+### 官网 {#si-chuan-shi-fan-da-xue-guan-wang}
+
+<Route author="nyaShine" example="/sicnu/www/23" path="/sicnu/www/:category?" paramsDesc={['分类ID']} radar="1">
```

Comment:
Please mention the default value of `category` if it's optional.
https://docs.rsshub.app/joinus/new-rss/add-docs